extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -161,3 +175,43 @@ class="jetpack-podcast-player__episode-link"
 
 	return ob_get_clean();
 }
+
+/**
+ * Given the color name, bock attributes and the CSS property,
+ * the function will return an array with the `class` and `style`
+ * HTML attributes to be used straight in the markup.
+ *
+ * @example
+ * $color = get_colors( 'secondary', $attributes, 'border-color'
+ *  => array( 'class' => 'has-secondary', 'style' => 'border-color: #333' )
+ *
+ * @param string $name     Color attribute name, for instance `primary`, `secondary`, ...
+ * @param array  $attrs     Block attributes.
+ * @param string $property Color CSS property, fo instance `color`, `background-color`, ...
+ * @return array           Colors array.
+ */
+function get_colors( $name, $attrs, $property ) {
+	$attr_color  = "{$name}Color";
+	$attr_custom = 'custom' . ucfirst( $attr_color );
+
+	$color        = isset( $attrs[ $attr_color ] ) ? $attrs[ $attr_color ] : null;
+	$custom_color = isset( $attrs[ $attr_custom ] ) ? $attrs[ $attr_custom ] : null;
+
+	$colors = array(
+		'class' => '',
+		'style' => '',
+	);
+
+	if ( $color || $custom_color ) {
+		$colors['class'] .= " has-{$name}";
```

Comment:
Do we need that leading whitespace here?

Reply:
yes, we are contacting two string.

Reply:
Connecting to an empty string though, no?

Reply:
this space together with `trim()` simplifies the process to handle these strings. We could use `array_push()` and at the end `implode()` it but I think it's almost the same.

Reply:
done!! sorry about this, thanks 🙏